### PR TITLE
Tidied the documentation for the Dashboard UI

### PR DESCRIPTION
**Pull request by jakexks**
As discussed with @bryk on [his fork](https://github.com/bryk/kubernetes/pull/1), I'm submitting this here.

I've rewritten the documentation for the new Dashboard UI so it flows slightly better.

### CODE REVIEW

File: docs/user-guide/ui.md [hunk 1]
```diff
@@ -58,42 +58,46 @@ found at https://github.com/kubernetes/dashboard/releases.
...
 ### Deploying applications
 
-With Dashboard you can deploy a replicated application using a simple form that guides through all
-required steps. All that is needed is a container image URI
-(e.g., on Google Container Registry or Docker Hub) and knowledge on what ports the image exposes.
-A replicated application that is deployed through the form is a replication controller plus optional
-service (if port mappings are specified).
+The Kuberntes Dashboard lets you create and deploy a Replication Controller with a simple wizard.
+You can simply provide the name for your application, the name of a Docker container (commonly 
+hosted on the Google Container Registry or Docker Hub,) and the target number of Pods you want deployed.
```
Comment (bryk) [inline docs/user-guide/ui.md]: Do we need comma after "Docker Hub"?

**Comment by bryk** [comment]
Tests are failing. Can you maybe fetch most recent changes, rebase and squash your commits into one? https://davidwalsh.name/squash-commits-git

**Comment by jakexks** [comment]
@bryk Rebased, removed the comma and stripped whitespace, hopefully this should pass the tests.

**Comment by bryk** [comment]
LGTM

**Comment by cheld** [comment]
@GertiPoppel

File: docs/user-guide/ui.md [hunk 2]
```diff
@@ -58,42 +58,46 @@ found at https://github.com/kubernetes/dashboard/releases.
...
 ![Kubernetes Dashboard welcome page](ui-dashboard-zerostate.png)
 
 ### Deploying applications
 
-With Dashboard you can deploy a replicated application using a simple form that guides through all
-required steps. All that is needed is a container image URI
-(e.g., on Google Container Registry or Docker Hub) and knowledge on what ports the image exposes.
-A replicated application that is deployed through the form is a replication controller plus optional
-service (if port mappings are specified).
+The Kuberntes Dashboard lets you create and deploy a Replication Controller with a simple wizard.
```
Comment (therc) [inline docs/user-guide/ui.md]: Typo

**Comment by bryk** [comment]
Thanks @jakexks for this contribution! :)

**Pull request merged**